README.md

```diff
@@ -36,7 +36,7 @@
 To get started quickly with a new project, run the command below:
 
 ```bash
-npx create-cloudflare@latest
+npm create cloudflare
```

Comment:
Nit: To avoid downloading the package twice if user has an older version. But it might be irrelevant here because mostly new user will try this.. 🤔 

```suggestion
npm create cloudflare@latest
```